staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -35,22 +35,30 @@ import (
 
 var (
 	explainLong = templates.LongDesc(i18n.T(`
-		List the fields for supported resources.
+		Describe fields and structure of various resources.
 
 		This command describes the fields associated with each supported API resource.
 		Fields are identified via a simple JSONPath identifier:
 
 			<type>.<fieldName>[.<fieldName>]
 
-		Add the --recursive flag to display all of the fields at once without descriptions.
 		Information about each field is retrieved from the server in OpenAPI format.`))
 
 	explainExamples = templates.Examples(i18n.T(`
 		# Get the documentation of the resource and its fields
 		kubectl explain pods
 
+		# Get all the fields in the resource
+		kubectl explain pods --recursive
+
+		# Get the explanation for deployment in supported api versions
+		kubectl explain deployments --api-version=v1beta2
```

Comment:
i tried apps/v1beta2 actually. I was going for v1 only seems like I changed in the end without notice. thanks for pointing it out.

Reply:
Also When i try for apps/v1beta2, i see the error:

error: couldn't find resource for "apps/v1beta2, Resource=deployments"

I understand it is because it's not supported in this k8s version. am i right?

Reply:
Probably, but then we should find a relevant example.